.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: node_js
 sudo: false
 install:
-  - npm install npm@2
-  - ./node_modules/.bin/npm install
+  - npm install -g npm@2
```

Comment:
There was an issue with npm not installing my modules properly and couldnt find the react-time-select. I think it was caused from travis caching the modules and the install not working properly